.examples/cron_example.go

```diff
@@ -142,10 +142,10 @@ func main() {
 			Payload: &anypb.Any{Value: []byte("ev 7s")},
 		})
 		cron.AddJob(ctx, etcdcron.Job{
-			Name:    "every-1s-then-expire-hadfh452erhh",
-			Rhythm:  "*/1 * * * * *",
-			TTL:     10,
-			Payload: &anypb.Any{Value: []byte("ev 1s then expires after 10s")},
+			Name:       "every-1s-then-expire-hadfh452erhh",
+			Rhythm:     "*/1 * * * * *",
+			Expiration: time.Now().Add(10 * time.Second),
```

Comment:
This is better to keep compatibility between Job and JobRecord. It means Dapr will need to calculate the Expiration instead of passing TTL - a minor breaking change IMO.

Reply:
Expiration is fine instead of TTL since it will be a fixed time period instead of a value that is decremented.